### PR TITLE
Set a note while creating an authorization

### PR DESCRIPTION
**Pull request by kdaigle**
Recently, the `note` attribute became a required part of personal access tokens. In it's current state, `octonaut` can't create an access token.

I decided to go with `Time.now` for the unique part of the token note because I thought it'd be the most helpful for the end-user. I'm not tied to using that at all.

:rocket:

### CODE REVIEW

File: lib/octonaut/commands/authorizations.rb [hunk 1]
```diff
@@ -7,7 +7,7 @@ module Octonaut
       password = ask("Enter your password:  ") { |q| q.echo = false }
 
       client = Octokit::Client.new :login => username, :password => password
-      authorization = client.create_authorization :scopes => []
+      authorization = client.create_authorization :scopes => [], :note => "Octonaut #{Time.now}"
```
Comment (pengwynn) [inline lib/octonaut/commands/authorizations.rb]: Nice. Might be helpful to put the HOST in here, too like hub does.

Comment (kdaigle) [inline lib/octonaut/commands/authorizations.rb]: If I understand that correctly, won't that just use the HOST of the GitHub instance? Meaning, it'll always be whatever instance you're viewing the note on?

Comment (pengwynn) [inline lib/octonaut/commands/authorizations.rb]: Was thinking something like:

``` ruby
>> require 'socket'
true
>> Socket.gethostname
"wynning.local"
```

Comment (kdaigle) [inline lib/octonaut/commands/authorizations.rb]: Ah, derp. That makes way more sense. Please hold... :phone:

**Pull request merged**